tools/replay/rp_visualization.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import sys
+import pygame
+import numpy as np
+import rerun as rr
+import cereal.messaging as messaging
+from openpilot.common.basedir import BASEDIR
+from openpilot.common.transformations.camera import DEVICE_CAMERAS
+from openpilot.tools.replay.lib.ui_helpers import (UP, Calibration,
+                                         get_blank_lid_overlay,
+                                         maybe_update_radar_points, plot_lead,
+                                         plot_model)
+from msgq.visionipc import VisionIpcClient, VisionStreamType
+
```

Comment:
what's this for?

Reply:
It has something to do with calibration. I just impirted it from ui.py. I'll check the code to see if it can be removed.

Reply:
It serves for checking if any new data is coming, if not it skips loop iteration which saves a lot of memory.